src/airport_connections_normalized.vl.js

```diff
@@ -1,5 +1,65 @@
 import * as vl from 'vega-lite-api';
 
+function generateVegaLiteSpec() {
```

Comment:
change this to `export default function generateVegaLiteSpec() {` for the tests to work